### ISSUE TITLE
About the parameter settings

### ISSUE THREAD

**Issue by yerui51** [1049451037/GCN-Align#2]
I am very interested in  your papers  and want to reproduce your experiment. I would like to ask about the settings of your parameters， including the learning rate, number of negative samples, number of iterations of structure and attribute. Thank you very much.  Eager for your reply!

**Comment by 1049451037** [comment]
Hi, thank you for your interest! The parameters are in the ***include/Config.py***. The parameters here are fine-tuned afterwords, which performs better than the paper, the parameters of the experiments are demonstrated in the paper.

**Comment by yerui51** [comment]
Hi, thanks for your reply. I have successfully run your code. When I read the code, I found a small error, in the get_sparse_tensor function, ind.append((sec, fir)), sec and fir，is their position reversed? . I would like to ask if my understanding is biased or your small mistake, thank you.

**Comment by 1049451037** [comment]
I re-checked it and I think it is right. In the paper we define that $a_ij$ is the effect of i-th entity to j-th entity. But here the entity matrix is left multiplied by the M matrix, which means $M_ij$ equals $a_ji$. Hope it helps!
